pixi.toml

```diff
@@ -79,7 +79,7 @@ test-ribasim-core = { cmd = "julia --project=core --eval 'using Pkg; Pkg.test()'
 test-ribasim-migration = { cmd = "pytest --numprocesses=4 -m regression python/ribasim/tests" }
 test-ribasim-core-cov = { cmd = "julia --project=core --eval 'using Pkg; Pkg.test(coverage=true, julia_args=[\"--check-bounds=yes\"])'", depends-on = [
     "generate-testmodels",
-] }
+], env = { SSL_CERT_DIR = "", JULIA_SSL_CA_ROOTS_PATH = "" } }
```

Comment:
Unrelated, but this is needed to be able to use branches in our Manifest.toml, found in #2046.

Reply:
Could this break other uses cases (TC?) and if needed, for branches, it would probably be required for all Julia related tasks, not just core-cov?

Reply:
No this is just when Pkg uses LibGit2 to checkout a branch. The other tasks that may need this already have this set. We can remove this again when https://github.com/JuliaLang/julia/pull/56924 is merged.